src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -324,7 +324,7 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
   getAllUsers(): Observable<any> {
     return Observable.combineLatest(
       this.userService.getUser(),
-      this.userService.getAllUsers()
+      this.workItemService.getCollaborators()
```

Comment:
Thanks a lot, I will first add `collaborators` to https://github.com/fabric8-ui/ngx-fabric8-wit/blob/master/src/app/models/space.ts#L27 and then consume it here ?